CODEOWNERS

```diff
@@ -0,0 +1,25 @@
+# Nonspecific maintainers
+#   @laanwj
+#   @sipa
+#   @fanquake
+
+# Build system maintainer
+/configure.ac         @theuni
```

Comment:
Even though @theuni is the person we'd like send these reviews to, based on my recent discussions with him, I don't think that is going to be most effective right now. I'd rather @dongcarl receive the build system review notifications (assuming he doesn't mind?)

Reply:
Will wait for his response, otherwise i'll comment these out.